code/game/gamemodes/miniantags/bot_swarm/swarmer.dm

```diff
@@ -423,6 +423,8 @@
 /obj/structure/lattice/catwalk/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
 	. = ..()
 	var/turf/here = get_turf(src)
+	if(!here)
```

Comment:
Line 424 calls the disintegrate stuff on root level. The order of things are incorrect here.

Reply:
Ah, so the `..()` call should go at the *end* of the proc then.

Reply:
I'm not sure the `if(!here)` check is needed. The only reason it would runtime before is because parent would run first, destroy it (thus moving it to null space), andddd then it tries to check for something that shouldn't really normally ever happen.

Reply:
That's true.. I will remove it then.